tests/rt.conf

```diff
@@ -22,10 +22,9 @@ RUN     | fv3_lndp
 RUN     | fv3_iau                                                                                                                 |                                         | fv3 | fv3_control
 RUN     | fv3_lheatstrg                                                                                                           |                                         | fv3 |
 
-# WW3 not working on Cheyenne in the past, need to check if it works now
-COMPILE | APP=ATMW SUITES=FV3_GFS_2017,FV3_GFS_2017_gfdlmp                                                                        | + wcoss_dell_p3 hera.intel orion.intel  | fv3 |
-RUN     | fv3_gfdlmprad                                                                                                           | + wcoss_dell_p3 hera.intel orion.intel  | fv3 |
-RUN     | fv3_gfdlmprad_atmwav                                                                                                    | + wcoss_dell_p3 hera.intel orion.intel  | fv3 |
+COMPILE | APP=ATMW SUITES=FV3_GFS_2017,FV3_GFS_2017_gfdlmp                                                                        | - jet.intel                             | fv3 |
```

Comment:
Bin is adding Jet for CPLD and DATM tests in #533. I think that means these ATMW tests could be added to jet also.

Reply:
I can turn these on, but I have no access to Jet to create the baselines for this, should I go ahead and turn these on?

Reply:
Yes, the new baselines can be created automatically on jet (and cheyenne also) so they can be turned on.